range.c

```diff
@@ -2637,6 +2655,7 @@ Init_Range(void)
     id_beg = rb_intern_const("begin");
     id_end = rb_intern_const("end");
     id_excl = rb_intern_const("excl");
+    id_plus = rb_intern_const("+");
```

Comment:
```suggestion
    id_plus = '+';
```

Reply:
@knu I am not sure about this suggestion, can you please explain what I am missing?.. 

It seems that it doesn’t work when I tried, and I am not sure why it should.

Reply:
@zverok: It's likely a undocumented and a micro-optimization but [it's used in `Integer#succ`](https://github.com/ruby/ruby/blob/v3_3_4/numeric.c#L3696-L3707) so it should be working here too.